src/FileCommandFacility.cpp

```diff
@@ -0,0 +1,262 @@
+/** @file FileCommands.cpp
+
+    Fixme: the local "streams" types supporting FileCommandSource are
+    general and could be useful broken out.
+
+    Fixme: need more imp files with http://, zeromq://, gRPC://, etc.
+
+ */
+
+#include "appfwk/CommandFacility.hpp"
+#include "appfwk/Issues.hpp"
+#include "appfwk/DAQModuleManager.hpp"
+
+#include <cetlib/BasicPluginFactory.h>
+#include <nlohmann/json.hpp>
+
+#include <fstream>
+#include <string>
+#include <memory>
+
+using namespace dunedaq::appfwk;
+
+using object_t = nlohmann::json;
+
+// Abstract the object level
+struct ObjectStream {
+    std::string name;
+    std::iostream& io;
```

Comment:
This is a bit dangerous, as it does not control the lifetime of the stream given in constructions. 
Can we do something safer?